test/functional/wallet_importmulti.py

```diff
@@ -584,12 +584,12 @@ def run_test(self):
         self.test_importmulti({"desc": descsum_create(desc),
                                "timestamp": "now",
                                "range": 1},
-                              success=True,
-                              warnings=["Some private keys are missing, outputs will be considered watchonly. If this is intentional, specify the watchonly flag."])
+                              success=True)
         for address in addresses:
```

Comment:
This isn't testing anything (the `address` loop variable in not being used, and the address tested is left over from the test above)

Reply:
@jnewbery  not sure what you mean, it calls `self.test_address(address, ...)` with each address to ensure they imported correctly from the xpriv after failing to import in the test before?